### DISCUSSION TITLE
How long will github reply a ticket?

### DISCUSSION THREAD

**Discussion by yiyuxin** [community/community#34850] [New to GitHub] [answered]
How long will github reply a ticket?

**Comment by yiyuxin** [comment]
Really bad experience！！！！！！

**Reply by MelphinDev** [reply]
Just wait patiently. They will reply. It takes time.

**Reply by cirosantilli** [reply]
Just as a datapoint, I got a reply in 42 days recently for "GitHub Actions is currently disabled for your account" https://github.com/cirosantilli/china-dictatorship/issues/1330

**Reply by Harshith-517** [reply]
@yiyuxin Did you get reply? If yes after how many days did you get reply?

**Comment by melvincwng** [comment, marked as answer]
@yiyuxin Hi Yuxin, from my own experience, usually GitHub Support would reply to tickets within a few days to a few weeks. 

The time required to answer your question depends on many things - e.g. the nature of the question, how much they need to investigate or coordinate with other teams, how many tickets they have to handle etc.

My suggestion is to patiently wait for them to respond especially if it's an account-specific or GitHub product-specific issue.

Alternatively, if the question could be answered by members of the community, feel free to ask away in another discussion thread.

Hope this clarifies and answers your question, take care 😄

**Reply by yiyuxin** [reply]
I’m really very grateful to you

**Reply by yiyuxin** [reply]
Have no other choice

**Comment by tbkyoto** [comment]
Still waiting after 13 days, regarding a question concerning our billing. So bad.

**Reply by ghost** [reply]
YES

**Reply by RanjanAstra** [reply]
Were you able to resolve the billing issue ?

**Reply by ozturkyusufziya** [reply]
Create a ticket and they will return if dont start spamming all community questions

**Comment by TrippleAWap** [comment]
it's been 3+ weeks and I'm still waiting..

**Comment by jayesh-xyz** [comment]
my account got suspended, I have to apply for jobs and I have no github to showcase. Not getting any resposne too!

**Reply by miguelcondesantos** [reply]
you got your account back, or this is the account that got suspended?

**Reply by vedantgupta23mc171-create** [reply]
hi , when did you got revert , can you pls help

**Comment by ngotanloi0709** [comment]
i can't create codespace due to billing issues, still waiting for response :<

**Comment by pwngrg** [comment]
Opened a ticket week ago and still waiting for response from the staff. Not a good experience so far.

**Comment by ozturkyusufziya** [comment]
I've been waiting for a response for 1 month. A week ago, I received a message saying "We have not forgotten about you..." but now I feel like I have been forgotten. From now on, I'll be tagging @github-staff in every discussion that has this issue. I guess you need to be Karen to get a problem solved.

**Comment by SQTX** [comment]
I've the same problem. I've been waiting for over 1 month and still nothing. @github-staff
[#3036531](https://support.github.com/ticket/3036531)

**Comment by nellimonix** [comment]
21 января создал тикет, сейчас 23 марта. Посчитайте сколько времени прошло. Поддержка гитхаба - это самое смешное изобретение человечества. Снесли мне мой аккаунт, 2 месяца ждать ради чего?

**Comment by barsikus007** [comment]
my account was flagged too (without any reason information)

I can't search code, I can't develop my OSS projects, I can't create issues (they are invisible for other world (and this is shadowed, I can see them, others not)), I even can't work with my team inside **private** repo...
and main issue, that I even don't know **why** I am suffering??

support page says it is 7 working days but how you can see it isn't right:
![image](https://github.com/user-attachments/assets/b6159dbc-e4eb-4995-bf46-f34763d7f90d)

why flagging is instant for my, old enough, account, when unflagging takes literal ages?

**Reply by barsikus007** [reply]
UPD: I was unbanned after 6 weeks of nothing

**Reply by ghost** [reply]
https://support.github.com/ticket/personal/0/3891813

**Comment by VortNs** [comment]
Reading similar topics gave me impression that unbanning process usually takes months. But here is my case: account was banned 18 Jul and unbanned 21 Jul (today), resulting in 3 days of waiting.
